examples/jit-trace/README.md

```diff
@@ -0,0 +1,8 @@
+To run this example, you will have to unzip the [MNIST data
+files](http://yann.lecun.com/exdb/mnist/) in `data/`.
+
+# Saving a trained model by tracing
+
+This tutorial illustrates how to serialize a model trained in Rust as a TorchScript program.
+First, we will train a model on the MNIST dataset and serialize it as a TorchScript program in Rust.
+Then, we will load and run it on the test dataset in Python.
```

Comment:
Looks good, maybe you could also add the shell commands to run for this as well as the expected outputs so that it's easier for interested folks to reproduce the whole setup.

Reply:
Thank you! I added more descriptions to README.